app/assets/javascripts/application.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * Created by jeka on 5/14/17.
+ */
+//= require_tree .
+//= require jquery
+//= require jquery_ujs
```

Comment:
please, be attentive about newline in the end. Check please all the files in this pull request